_opensearch/remote.md

```diff
@@ -11,6 +11,51 @@ Remote-backed storage is an experimental feature. Therefore, we do not recommend
+
+Thus, a translog contains all operations that have not yet been written to disk. 
+
+## Segment replication and remote-backed storage
+
+When neither segment replication nor remote-backed storage is enabled, OpenSearch uses document replication. In document replication, when a write request lands on the primary shard, the request is indexed to Lucene and stored in the translog as a single unit of transaction for durability. After this, the request is sent to the replicas, where, in turn, it is indexed to Lucene and stored in the translog. 
+
+With segment replication, the segments are created on the primary shard only and then copied to all replicas. The replicas do not index a request to Lucene, but they do create and maintain a translog.
+
+With remote-backed storage, when a write request lands on the primary shard, the request is indexed to Lucene on primary shard only, and then stored in the translog. The translogs are fsynced to local disk and then uploaded into remote store. OpenSearch does not send the write request to the replicas, but rather performs a primary term validation. Primary term validation entails confirming that the request originator shard is still the primary shard. This is necessary to ensure that the acting primary shard fails in case the acting primary is isolated and unaware of the cluster manager electing a new primary.
+
+## Translog settings
+
+Without remote-backed storage, indexing operations are only persisted to disk when the translog is fsynced. Therefore, any data that has not been written to disk can potentially be lost. 
+
+The `index.translog.durability` setting controls how frequently OpenSearch fsyncs the translog to disk:
+
+- By default, `index.translog.durability` is set to `request`. This means that fsync happens after every request, and all acknowledged write requests persist in case of failure.
+
+- If you set` index.translog.durability` to `async`, fsync happens periodically at the specified `sync_interval` (5 seconds by default). The fsync operation is asynchronous, so acknowledge is sent without waiting for fsync. Consequently, all acknowledged writes since the last commit are lost in case of failure.
+
+With remote-backed storage, the translog is not only fsynced to disk, but also uploaded into a remote store.
+
+## Refresh-level and request-level durability
+
+The remote store feature supports two levels of durability:
+
+- Refresh-level durability: Segment files are uploaded to remote store after every refresh. Set the `remote_store` flag to `true` to achieve refresh-level durability. Commit-level durability is inherent, and uploads are asynchronous.
+
+  If you need to activate a refresh manually, you can use the `_refresh` API. For example, to refresh the `my_index` index, use the following request:
+   
+  ```json
+  POST my_index/_refresh
+  ```
+
+  To refresh all indexes, use the following request:
+
+  ```json
+  POST /_refresh
+  ```
```

Comment:
do we need to provide this ?

Reply:
Yes, may be refresh all indexes part is not needed. But the refresh of specific index is definitely something we need when we have refresh level durability.